doc/source/train/examples/lightning/lightning_mnist_example.ipynb

```diff
@@ -123,7 +110,9 @@
     "            nn.ReLU(),\n",
     "        )\n",
     "        self.lr = lr\n",
-    "        self.accuracy = Accuracy()\n",
+    "        self.accuracy = Accuracy(task=\"multiclass\", num_classes=10)\n",
```

Comment:
CAn you include the package installation instructions at the top of this example?